src/navigators/AbstractChildWindow.ts

```diff
@@ -87,11 +90,24 @@ export abstract class AbstractChildWindow implements IWindow {
         this._disposeHandlers.clear();
     }
 
-    protected static _notifyParent(parent: Window, url: string, keepOpen = false, targetOrigin = window.location.origin): void {
-        parent.postMessage({
+    protected static _notifyParent(parent: Window | null, url: string, keepOpen = false, targetOrigin = window.location.origin): void {
+        const msgData: MessageData = {
             source: messageSource,
             url,
             keepOpen,
-        } as MessageData, targetOrigin);
+        };
+        if (parent) {
+            parent.postMessage(msgData, targetOrigin);
+        } else {
+            const logger = new Logger("_notifyParent");
+            logger.debug("No parent. Using BroadcastChannel.");
```

Comment:
When i was debugging this i immediately added a log in the `if (parent)` code path as well. Please do so too.
Something like:
```
...
const logger = new Logger("_notifyParent");
if (parent) {
    logger.debug("With Parent. Using parent.postMessage.");
} ...
```

Reply:
Done.